source/agora/test/Base.d

```diff
@@ -986,11 +993,14 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
         break;
     }
 
+    immutable const_params =
+        (params !is null) ? params : new immutable(ConsensusParams)();
```

Comment:
Technically this is not const anymore. Maybe:

```suggestion
    immutable cons_params =
        (params !is null) ? params : new immutable(ConsensusParams)();
```

So it refers to consensus, rather than "constant".

Reply:
Right, it should have the meaning of the `consensus`.